.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
```

Comment:
**Suggestion**: Specify the exact version of ubuntu-latest to use for the runs-on property to ensure consistent workflow execution environments. <br> **Code Suggestion**: 
 ```
 - runs-on: ubuntu-20.04
 ```